homer.root/usr/share/clearwater/cassandra-schemas/homer.sh

```diff
@@ -16,7 +17,24 @@ then
     /usr/share/clearwater/bin/poll_cassandra.sh --no-grace-period
   done
 
-  echo "CREATE KEYSPACE homer WITH REPLICATION = {'class': 'SimpleStrategy', 'replication_factor': 2};
+  replication_str="{'class': 'SimpleStrategy', 'replication_factor': 2}"
+
+  # If local_site_name and remote_site_names are set then this is a GR
```

Comment:
Good idea - I'll move it to the clearwater-cassandra repo (but still install the Bash file to /usr/share/clearwater/cassandra-schemas)